metricbeat/module/elasticsearch/enrich/data.go

```diff
@@ -38,17 +38,34 @@ var (
 		},
 		"remote_requests": s.Object{
 			"current": c.Int("remote_requests_current"),
-			"total":   c.Int("remote_requests_total"),
+			"count":   c.Int("remote_requests_total"),
 		},
 		"executed_searches": s.Object{
-			"total": c.Int("executed_searches_total"),
+			"count": c.Int("executed_searches_total"),
 		},
 	}
+
+	task = s.Schema{
+		"id":     c.Int("id"),
+		"type":   c.Str("type"),
+		"action": c.Str("action"),
+		"time": s.Object{
+			"start": s.Object{
+				"ms": c.Int("start_time_in_millis"),
+			},
+			"running": s.Object{
+				"nano": c.Int("running_time_in_nanos"),
+			},
+		},
+		"cancellable":    c.Bool("cancellable"),
+		"parent_task_id": c.Str("parent_task_id"),
+		// "headers": for reviewers -> ...What's the expected output of this? Maybe it's not necessary to add it
+	}
 )
 
 type response struct {
 	ExecutingPolicies []map[string]interface{} `json:"executing_policies"`
-	CoordinatorStats  []map[string]interface{} `json:"coordinator_stats"`
+	CoordinatorStats []map[string]interface{} `json:"coordinator_stats"`
```

Comment:
Surprised that the linter is okay with this change.

Reply:
Fixed! Thank you for spotting this one